lib/presentation/widget/shared_space/file_surfing/workgroup_detail_files_widget.dart

```diff
@@ -64,14 +69,16 @@ class _WorkGroupDetailFilesWidgetState extends State<WorkGroupDetailFilesWidget>
           _workgroupNavigatorKey,
           widget.sharedSpaceNode,
           widget.onBackClickedCallback,
+          nodeSurfingType: widget.nodeSurfingType,
         ),
         Align(
           alignment: Alignment.bottomCenter,
           child: SizedBox(
             height: 76,
             child: Stack(
               children: [
-                Align(
+                widget.nodeSurfingType == NodeSurfingType.normal
```

Comment:
I think we should seperate whole widget, it is more clear and updateable in future in case we have more type (just my opinion)
```
switch (type) {
    case NodeSurfingType.normal:
        return _buildDocumentNormalItem();
    case NodeSurfingType. destinationPicker:
        return _buildDestinationPickerItem();
}
```